docker/README.md

```diff
@@ -0,0 +1,47 @@
+# Docker
+
+This Task builds source into a container image using [Docker](https://github.com/docker/). This command executes the directives in the `Dockerfile` to assemble a container image, then pushes that image to a container registry.
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/docker/docker.yaml
```

Comment:
Once the task is added in the task directory could you please update the path as well